blazor/diagram/migration-classic-to-native.md

```diff
@@ -0,0 +1,577 @@
+---
+layout: post
+title: Migration from SfDiagram(classic) to SfDiagramComponent | Syncfusion
+description: Learn here all about how to migrate the SfDiagram to SfDiagramComponent in Syncfusion Blazor Diagram component and more.
+platform: Blazor
+control: Diagram Component
+documentation: ug
+---
+
+# Migration from SfDiagram(classic) to SfDiagramComponent control
+
+This article describes the API migration process of SfDiagramComponent from SfDiagram(classic).
+
+## NuGet packages
+
+| Package for SfDiagram | Package for SfDiagramComponent |
+| -------- | -------- |
+| [Syncfusion.Blazor.Diagrams](https://www.nuget.org/packages/Syncfusion.Blazor.Diagrams/#%22%22) | [Syncfusion.Blazor.Diagram](https://www.nuget.org/packages/Syncfusion.Blazor.Diagram/#%22%22) |
+
+## Namespace changes
+
+| Control | Namespace for SfDiagram | Namespace for SfDiagramComponent |
+| -------- | -------- | -------- |
+| Diagram | Syncfusion.Blazor.Diagrams | Syncfusion.Blazor.Diagram |
+| SymbolPalette | Syncfusion.Blazor.Diagrams | Syncfusion.Blazor.Diagram.SymbolPalette |
+| Overview | Syncfusion.Blazor.Diagrams | Syncfusion.Blazor.Diagram.Overview |
+
+## Advantages
+
+A `SfDiagramComponent` blazor component which provides better performance than the `SfDiagram` blazor control in Blazor WebAssembly Application.
```

Comment:
A `SfDiagramComponent` blazor component performs better than the `SfDiagram` blazor control in the Blazor WebAssembly Application.